bool get_wpts
---
float64[] latitude
float64[] longitude
int16 num_wpts
bool success
